lmeval-scenarios.adoc

```diff
@@ -61,6 +61,8 @@ env:
 
 == Custom Unitxt Card
 
+It is also possible to run evaluations using custom unitxt cards. To do so, add the custom unitxt card in JSON format to the LMEvalJob YAML.
```

Comment:
Consider using second-person imperative mood here:

You can also run evaluations using custom unitxt cards. To do this, include the custom unitxt card in JSON format within the LMEvalJob YAML.